### ISSUE TITLE
C extension should be configured via INI

### ISSUE THREAD

**Issue by Addvilz** [maxmind/MaxMind-DB-Reader-php#18]
It would be nice for it to be possible to set C extension's default database path in INI file, to allow system wide configuration for the extension, as it was done for the old extension.

**Comment by komapa** [comment]
Yes, this is a big regression from the legacy GeoIP extension: http://php.net/manual/en/geoip.configuration.php#ini.geoip.custom-directory
